bugbug/model.py

```diff
@@ -285,6 +285,24 @@ def print_feature_importances(
                 end="\n\n",
             )
 
+    def save_feature_importances(self, important_features, feature_names):
+        # Returns a JSON-encodable dictionary that can be saved in the metrics
+        # report
+        top_feature_names = [
+            feature_names[int(index)]
+            for importance, index, is_pos in important_features["average"]
+        ]
+
+        feature_report = {}
+
+        for i, feature_name in enumerate(top_feature_names):
+            for class_name, imp_values in important_features["classes"].items():
+                feature_report.setdefault(feature_name, {})[class_name] = float(
```

Comment:
For the check that we will implement in the future, it might be better to have the dict be:
class_name -> feature_name -> value
rather than
feature_name -> class_name -> value

Since what we want to check is whether a feature which was not important before suddenly became important, or whether a feature which was important suddenly became not important.

Reply:
Your call here, I don't have a preference ;)